src/page/Home.css

```diff
@@ -0,0 +1,37 @@
+.home-logo {
+    font-size: 70px;
+    color: #04A9EB;
+    font-family: Futura;
```

Comment:
We should probably have a standard fallback font in case `Futura` doesn't exist in user's system. Ideally we should load our fonts in the `package.json` to make sure they exist, but not sure if there is a proper package for `Futura` font

Reply:
@onursumer can I replace the text to a picture?